content/open-source/concepts/webpack/webpack.md

```diff
@@ -0,0 +1,21 @@
+---
+Title: 'Webpack'
+Description: 'Webpack is a JavaScript module bundler that can be run on the Linux, macOS, and Windows operating systems.'
+Subjects:
+  - 'Developer Tools'
+  - 'Web Development'
+Tags:
+  - 'JavaScript'
+  - 'Modules'
+  - 'Node'
+  - 'Open Source'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/computer-science'
+---
+
+**Webpack** is a free and open-source [JavaScript module](https://www.codecademy.com/resources/docs/javascript/modules) that can also be applied to [HTML](https://www.codecademy.com/resources/docs/html) and [CSS](https://www.codecademy.com/resources/docs/css). Webpack can be run on the [Linux](https://www.codecademy.com/resources/docs/open-source/linux), macOS, and Windows operating systems.
+
+As a module bundler, Webpack can be used with `webpack.config.js` to generate dependency graphs, and define rules for tasks performed on a site. Primarily it is used by developers to load, transform, and package modules for in-browser use.
```

Comment:
As a module bundler, Webpack can be used with `webpack.config.js` to generate dependency graphs and define rules for tasks performed on a site. Primarily it is used by developers to load, transform, and package modules for in-browser use.

Reply:
Removed comma.